### ISSUE TITLE
VAE-GAN for tissue targeted sequences

### ISSUE THREAD

**Issue by digivi13** [samsinai/VAE_protein_function#2]
What are your thoughts on extending your VAE architecture to a VAE-GAN? I think a higher order similarity metric, and the use of two loss functions may increase the efficacy of this application in your case. If applied to AAV research, could you potentially identify capsid sequences of high stability and feed them to the VAE, then instead of feeding the discriminator that same training data, you use another dataset or subset of sequences confirmed to show specificity for a particular tissue, this way you may be able to generate sequences of high stability and specificity to deliver your target. 
 
Also, do you know where I may find some public AAV data? I may give this a go. 

Here is the architecture I was thinking of experimenting with:

<img width="591" alt="Screen Shot 2019-04-08 at 8 07 43 PM" src="https://user-images.githubusercontent.com/33264428/55764610-55775e00-5a3a-11e9-9c1b-e7ec0738316a.png">
